### PR TITLE
xorg: Add information about `startx`.

### PR DESCRIPTION
**Pull request by flexibeast**
Addresses #239.

### CODE REVIEW

File: src/config/graphical-session/xorg.md [hunk 1]
```diff
@@ -82,3 +82,19 @@ Section "InputClass"
...
+
+```
+exec /bin/i3
+```
+
+Then call `startx` to start an i3 session:
+
+```
+$ startx
+```
```
Comment (ericonr) [inline src/config/graphical-session/xorg.md:100]: This feels like duplicated instructions. Perhaps just the "Then call..." line? And in this case, I'd argue that it shouldn't be a man pages link either.

Comment (flexibeast) [reply src/config/graphical-session/xorg.md]: Gah, sorry about that - i agree, on both points. :-) i've removed the example command from here, and modified #248.

**Comment by ericonr** [comment]
I think we can merge this one. Do you want to do anything else with it?

**Comment by flexibeast** [comment]
i'm happy to merge this as-is. But CI is still failing. :-(

**Comment by flexibeast** [comment]
Okay, CI is now passing, but i'll wait for the owner of #239 to let us know if there's anything else they think might be added.

**Comment by flexibeast** [comment]
Actually, since this PR hasn't yet been merged, i'll take the opportunity to use it to address #276. :-)

**Comment by flexibeast** [comment]
@ericonr: PR updated. Could you please review this and let me know if you think it's ready to merge?

**Comment by ericonr** [comment]
Sorry for the delay! I think it's good, and does a good job of helping with the DE specific stuff.

**Comment by flexibeast** [comment]
@ericonr: Thanks!

**Pull request merged**